src/utils/api.js

```diff
@@ -13,9 +13,17 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import liskJS from 'lisk-js';
+import lisk from 'lisk-js';
```

Comment:
Did we consider `liskElements` or just `elements`? @Tosch110 what are your thoughts?

Reply:
yes I was thinking to change it to `elements`, but not officially changed name.
so, I settled in kinda middle...

Reply:
As discussed, we'll leave it as it is for now.